src/System.Data.SqlClient/src/System/Data/SqlClient/SqlBulkCopyColumnMappingCollection.cs

```diff
@@ -104,50 +72,48 @@ private void AssertWriteAccess()
             }
         }
 
-        public void Clear()
+        new public void Clear()
         {
             AssertWriteAccess();
-            _list.Clear();
+            base.Clear();
         }
 
-        public bool Contains(SqlBulkCopyColumnMapping value) => _list.Contains(value);
+        public bool Contains(SqlBulkCopyColumnMapping value) => InnerList.Contains(value);
 
-        public void CopyTo(SqlBulkCopyColumnMapping[] array, int index) => _list.CopyTo(array, index);
+        public void CopyTo(SqlBulkCopyColumnMapping[] array, int index) => InnerList.CopyTo(array, index);
 
         internal void CreateDefaultMapping(int columnCount)
         {
             for (int i = 0; i < columnCount; i++)
             {
-                _list.Add(new SqlBulkCopyColumnMapping(i, i));
+                InnerList.Add(new SqlBulkCopyColumnMapping(i, i));
             }
         }
 
-        public IEnumerator GetEnumerator() => _list.GetEnumerator();
-
-        public int IndexOf(SqlBulkCopyColumnMapping value) => _list.IndexOf(value);
+        public int IndexOf(SqlBulkCopyColumnMapping value) => InnerList.IndexOf(value);
 
         public void Insert(int index, SqlBulkCopyColumnMapping value)
         {
             AssertWriteAccess();
-            _list.Insert(index, value);
+            InnerList.Insert(index, value);
         }
 
         public void Remove(SqlBulkCopyColumnMapping value)
         {
             AssertWriteAccess();
-            _list.Remove(value);
+            InnerList.Remove(value);
         }
 
-        public void RemoveAt(int index)
+        new public void RemoveAt(int index)
         {
             AssertWriteAccess();
-            _list.RemoveAt(index);
+            base.RemoveAt(index);
         }
 
         internal void ValidateCollection()
         {
             MappingSchema mappingSchema;
-            foreach (SqlBulkCopyColumnMapping a in _list)
+            foreach (SqlBulkCopyColumnMapping a in this)
```

Comment:
This is going to allocate an enumerator where it didn't allocate one before.  But I'm assuming that level of perf optimization here isn't important, given the scenarios?